material/notebooks/Transfer_learning.ipynb

```diff
@@ -0,0 +1,291 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transfer learning \n",
+    "\n",
+    "In this notebook, I will use Transfer learning in our problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from keras import applications\n",
+    "import keras\n",
+    "from keras.preprocessing.image import ImageDataGenerator\n",
+    "from keras import optimizers\n",
+    "from keras.models import  Model \n",
+    "from keras.layers import Dropout, Flatten, Dense, GlobalAveragePooling2D\n",
+    "from keras import backend as k \n",
+    "import os\n",
+    "from PIL import Image\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I copy $\\textbf{10%}$ of the image to a new folder and this will be my new training set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data_dir = \"../data/sample_new/train\"\n",
+    "validation_data_dir = \"../data/validation\"\n",
```

Comment:
Mimis, it seems that this validation folder is never created from the current code that we are working with. In Kaggle, there is only train and test files, so my guess is that validation is a subset of the train set.

I suggest to first push the code that creates the validation data set and then we can proceed with this pull request.

Reply:
It is the responsibility of each of us to download the dataset with the provided `Download.py` file. There is even an example in the `Download.ipynb` notebook for the train and test sets. Just add two lines to get the validation set and feel free to push this in a separate PR if you feel other collaborators may find it useful